backend/.gitignore

```diff
@@ -10,3 +10,5 @@
 # cache
 __pycache__
 
+node_modules/
+
```

Comment:
should not be needed to be added. if u have node_modules in your backend folder, you might have accidentally ran `yarn install` in it; you can delete node_modules in the backend if you have one in your local